test/behaviour/graql/GraqlSteps.java

```diff
@@ -128,238 +136,226 @@ public void graql_delete_throws_exception(String exception, String deleteQuerySt
-//            numericAnswers = tx.execute(graqlQuery.asMatchAggregate()).get();
-//        } else if (graqlQuery instanceof GraqlMatch.Group) {
-//            answerGroups = tx.execute(graqlQuery.asMatchGroup()).get();
-//        } else if (graqlQuery instanceof GraqlMatch.Group.Aggregate) {
-//            numericAnswerGroups = tx.execute(graqlQuery.asMatchGroupAggregate()).get();*/
-//        } else {
-//            throw new ScenarioDefinitionException("Only match and insert supported for now");
-//        }
+    @When("get answers of graql insert")
+    public void get_answers_of_graql_insert(String graqlQueryStatements) {
+        final GraqlInsert graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements));
+        clearAnswers();
+        answers = tx().query().insert(graqlQuery).collect(Collectors.toList());
+    }
+
+    @When("get answers of graql match")
+    public void graql_match(String graqlQueryStatements) {
+        final GraqlMatch graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatch();
+        clearAnswers();
+        answers = tx().query().match(graqlQuery).collect(Collectors.toList());
     }
 
     @When("graql match; throws exception")
     public void graql_match_throws_exception(String graqlQueryStatements) {
-        // TODO
+        assertThrows(() -> graql_match(graqlQueryStatements));
+    }
+
+    @When("get answer of graql match aggregate")
+    public void graql_match_aggregate(String graqlQueryStatements) {
+        final GraqlMatch.Aggregate graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchAggregate();
+        clearAnswers();
+        numericAnswer = tx().query().match(graqlQuery).get();
+    }
+
+    @When("get answers of graql match group")
+    public void graql_match_group(String graqlQueryStatements) {
+        final GraqlMatch.Group graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchGroup();
+        clearAnswers();
+        answerGroups = tx().query().match(graqlQuery).collect(Collectors.toList());
+    }
+
+    @When("get answers of graql match group aggregate")
+    public void graql_match_group_aggregate(String graqlQueryStatements) {
+        final GraqlMatch.Group.Aggregate graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchGroupAggregate();
+        clearAnswers();
+        numericAnswerGroups = tx().query().match(graqlQuery).collect(Collectors.toList());
     }
 
     @Then("answer size is: {number}")
     public void answer_quantity_assertion(int expectedAnswers) {
-        // TODO
-//        assertEquals(
-//                String.format("Expected [%d] answers, but got [%d]", expectedAnswers, answers.count()),
-//                expectedAnswers, answers.count());
+        assertEquals(String.format("Expected [%d] answers, but got [%d]", expectedAnswers, answers.size()),
+                     expectedAnswers, answers.size());
     }
 
-    @Then("concept identifiers are")
```

Comment:
This step is no longer used. Answer concept matching was simplified to just `uniquely identify answer concepts` a while back.